miio/exceptions.py

```diff
@@ -32,3 +32,7 @@ def __init__(self, error):
 
 class RecoverableError(DeviceError):
     """Exception communicating an recoverable error delivered by the target device."""
+
+
+class ViomiVacuumException(DeviceException):
```

Comment:
Please move this to `viomivacuum.py`, this file is only used for exceptions common for the whole project.

Reply:
done